source/slang/slang-emit-cpp.cpp

```diff
@@ -2476,6 +2508,33 @@ bool CPPSourceEmitter::tryEmitInstExprImpl(IRInst* inst, const EmitOpInfo& inOut
             }
             return true;
         }
+        case kIROp_MakeExistential:
+        case kIROp_MakeExistentialWithRTTI:
+        {
+            auto rsType = cast<IRComPtrType>(inst->getDataType());
+            m_writer->emit("ComPtr<");
+            m_writer->emit(getName(rsType->getOperand(0)));
```

Comment:
Don't know if it's covered in this PR, but the other irritating issue is wiring between implementations, when multiply derived. It's not enough to derive with a new interface if there are shared functions in the super interface shared by both, they need to be 'wired' to call the existing implementation.